src/clients/schemas.ts

```diff
@@ -40,6 +40,29 @@ const JobDetailsSchema = z.object({
   }),
 });
 
+const FlakyTestSchema = z.object({
+  flaky_tests: z.array(
+    z.object({
+      // time_wasted: z.number(),
+      // workflow_created_at: z.string(),
+      // workflow_id: z.string().nullable(),
+      // classname: z.string(),
+      // pipeline_number: z.number(),
+      // workflow_name: z.string(),
+      // test_name: z.string(),
+      // job_name: z.string(),
+      job_number: z.number(),
+      // times_flaked: z.number(),
+      // source: z.string(),
+      // file: z.string(),
+    }),
+  ),
+  total_flaky_tests: z.number(),
+});
```

Comment:
note: the api docs seem to be incorrect here: https://circleci.com/docs/api/v2/index.html#tag/Insights/operation/getFlakyTests

they have the response coming back `kebab-case`, but it seems to come back `snake_case` like the rest of our apis. we should probably change the docs

commented out the parts of the response that we don't actually use in the schema. will remove those before merging